addons/actions/package.json

```diff
@@ -21,22 +21,19 @@
   },
   "homepage": "https://github.com/storybooks/storybook/tree/master/addons/actions",
   "devDependencies": {
-    "@storybook/addons": "^3.0.0-rc.0",
```

Comment:
Unrelated to docs, but we'll let it slide 😉

Reply:
@ndelangen Is there a chance that you can remember why was it needed to move `@storybook/addons` frop peerDeps to direct deps?

Reply:
Absolutely 0 chance of me remembering, sorry.

Though I think it makes sense, and without lock-files would work great.
Having these lockfiles really messes up the singleton module method, which lib/addons depends on to function.

Reply:
> without lock-files would work great

But it's probably the only reason users have to use the same **patch** version for all the `@storybook` dependencies